test/client_test.dart

```diff
@@ -196,12 +196,12 @@ void main() {
 
     test('Update user', () async {
       final response = await client.updateUser(
-        UserAttributes(data: {
-          'hello': 'world',
```

Comment:
This line is deleted, but required for the test.

Reply:
Yup, thanks for spotting it!